entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,193 @@
+        storage_icon: '/common/images/storage.png',
+        music_icon: '/common/images/music.png',
+        settings_icon: '/common/images/settings.png',
+        view_icon: '/common/images/view_icon.png',
+        add_icon: '/common/images/add_icon.png',
+        pink_button: '/common/images/pink_button.png',
+
+        //      T E X T
+        storage_settings:'Storage Settings',
+        dinner_text:'Dinner Recepies for you',
+        all_recp_text:'All Recepies',
+        food_remain: 'Food Remaining',
+        home_txt: "Home",
+        storage_txt: "Storage",
+        music_txt: "Music",
+        settings_txt: "Settings",
+        audio_assist1:"How can I help you?",
+        audio_assist2:"Tap to Speak",
+        your_storage:"Your Storage",
+        view_text:"View",
+        add_text:"Add",
+        door_text:"Refrigerator Doors",
+        time:"",
+        day_date:"",
+
+        //      Device Details
+        username: 'UserName',
+        location_city:"Bangalore",
+        //day_date:"Wednesday, 17 Dec",
+        temp_red:"30°",
+
+        //      Music Information
+        music_cover_image:'/common/images/dreams_img.png',
+        song_name: 'Dreams',
+        song_genre: 'Romantic',
+
+        //      Food Remaining tab
+        vegetable_percentage:"72%",
+        fruits_percentage:"50%",
+        meat_percentage:"20%",
+
+        //      Storage Setting Items
+        st_items :[],
+
+        //      Refrigeator Settings
+        door_status:"Locked",
+        door_status_toggle:"Unlock",
+        superfreeze_toggle_txt:"On",
+        superfreeze_toggle_txt_bool:"",
+        bottom_box_left_toggle_text: "Freeze",
+        bottom_box_right_toggle_text: "Fresh",
+        crisp_fresh_toggle_txt:"On",
+        crisp_fresh_toggle_txt_bool:"",
+
+
+        //      Recepie 1 Information
+        recp_1_image: '/common/images/vada.png',
+        recp_1_name:"Vada",
+        recp_1_time:"30 Mins",
+        recp_1_type:"Starters",
+        recp_1_user_reaction:"false",
+
+        //      Recepie 2 Information
+        recp_2_image: '/common/images/meals.png',
+        recp_2_name:"Meals",
+        recp_2_time:"45 Mins",
+        recp_2_type:"Main Course",
+        recp_2_user_reaction:"true",
+
+        //      Recepie 3 Information
+        recp_3_image: '/common/images/dosa.png',
+        recp_3_name:"Dosa",
+        recp_3_time:"15 Mins",
+        recp_3_type:"Main Course",
+        recp_3_user_reaction:"false",
+
+    },
+    onInit() {
+        setInterval(this.date_time_calc,1000);
+
+        if(this.superfreeze_toggle_txt == "On"){
+            this.superfreeze_toggle_txt_bool="true";
+        }
+        else{
+            this.superfreeze_toggle_txt_bool="false";
+        }
+
+        if(this.crisp_fresh_toggle_txt == "On"){
+            this.crisp_fresh_toggle_txt_bool="true";
+        }
+        else{
+            this.crisp_fresh_toggle_txt_bool="false";
+        }
+
+        this.getRecipeList(this);
```

Comment:
1. Also go through factory design pattern of JS (https://www.dofactory.com/javascript/design-patterns/abstract-factory) and try to define similar factories in separate javascript file (DummyFactory, ServerFactory)... and getRecipeList can get data from DummyFactory
2. similar way for all other possible data including settings